packages/ckeditor5-autoformat/tests/autoformat.js

```diff
@@ -410,6 +410,146 @@ describe( 'Autoformat', () => {
 			expect( getData( model ) ).to.equal( '<paragraph>**foobar**[]</paragraph>' );
 		} );
 
+		it( 'should not format if the plugin is not enabled', () => {
+			editor.plugins.get( 'Autoformat' ).forceDisabled( 'Test' );
+
+			setData( model, '<paragraph>**foobar*[]</paragraph>' );
+
+			model.change( writer => {
+				writer.insertText( '*', doc.selection.getFirstPosition() );
+			} );
+
+			expect( getData( model ) ).to.equal( '<paragraph>**foobar**[]</paragraph>' );
+		} );
+
+		describe( 'with code element', () => {
+			it( 'should not format inside', () => {
+				// Test *.
```

Comment:
These are OK to read when you browse tests. But when one of them fails, they are useless because there are many (similar) assertions within a single failing `it()` and you don't know which one it is. You can pass an [optional message to `expect()`](https://www.chaijs.com/api/bdd/#method_equal) so when an assertion fails, it is clear which one. [Check out how I did this in some tests](https://github.com/ckeditor/ckeditor5/commit/33e9b37c3808e53cf1e6025fb7ed250b393d9f4b#diff-e725861539867051799a155cdf02b85bR26-R49).

Alternatively, you could create multiple `it()` and boost our test stats :P